docs/how_to_publish_to_cmr_with_earthdata_varinfo.ipynb

```diff
@@ -161,22 +159,16 @@
    "source": [
     "from cmr import CMR_UAT\n",
     "\n",
-    "from varinfo.cmr_search import (get_granules, get_granule_link, \n",
-    "                                download_granule)\n",
-    "\n",
     "from varinfo import VarInfoFromNetCDF4\n",
-    "\n",
-    "from varinfo.umm_var import (get_all_umm_var, get_umm_var, publish_all_umm_var,\n",
-    "                             publish_umm_var)"
+    "from varinfo.cmr_search import download_granule, get_granules, get_granule_link\n",
+    "from varinfo.umm_var import get_all_umm_var, publish_all_umm_var, publish_umm_var"
```

Comment:
These changes just strip out the stuff that is only used in workflow 3 (single UMM-Var record), as those are now imported there, so each workflow is self-contained.